.github/workflows/python-package.yml

```diff
@@ -66,6 +111,8 @@ jobs:
         run: |
           cd repository
           python setup.py install
+          python -c "import sphinx_typo3_theme; print('Version: ' + sphinx_typo3_theme.__version__)"
+          python -c "import sphinx_typo3_theme; print('Version Full: ' + sphinx_typo3_theme.__version_full__)"
```

Comment:
sphinx_typo3_theme.__version_full__ doesn exist anymore. It used to be the same as __version__. Please drop that line.